src/assets/css/custom.scss

```diff
@@ -0,0 +1,136 @@
+.btn-platform {
+  background-color: #ffff;
+  color: #6a7888;
+  padding: 8px 8px 2px 8px;
+}
+
+.btn-platform:hover {
+  color: #6a7888;
+}
+
+.page-dropdown-menu {
+  min-width: 333px;
+}
+
+/* Setup some colors */
+:root {
+  --tabs-blue: #1572c2;
+  --tabs-light: #c3c9cf;
+  --tabs-grey: #6a7888;
+  --tabs-border: #cdddee;
+  --tabs-scroll-bg: #ebeef2;
+  --tabs-white: #ffffff;
+}
+
+/* Override Bootstrap default tab styles */
+.platform-tab {
```

Comment:
use the default class `.nav-tabs`, Product give us the green light to apply the style to all the tabs in the platform